scripts/spec/target_spec.rb

```diff
@@ -13,11 +13,11 @@
   end
 
   it "#imagefactory_type" do
-    expect(described_class.new("openstack").imagefactory_type).to eql "openstack-kvm"
+    expect(described_class.new("vhd").imagefactory_type).to eql "vpc"
   end
 
   it "#file_extension" do
-    expect(described_class.new("openstack").file_extension).to eql "qc2"
+    expect(described_class.new("vhd").file_extension).to eql "vpc"
```

Comment:
Why were these tests changed?  Not that it *really* matters, but I was curious why you changed it.